data/maps/LavaridgeTown_Gym_1F/scripts.inc

```diff
@@ -3,205 +3,205 @@ LavaridgeTown_Gym_1F_MapScripts:: @ 81FE6F4
 	.byte 0
 
 LavaridgeTown_Gym_1F_OnTransition: @ 81FE6FA
-	call LavaridgeTown_Gym_1F_EventScript_1FE705
-	call LavaridgeTown_Gym_1F_EventScript_1FE757
+	call LavaridgeTown_Gym_1F_EventScript_SetTrainerTempVars
+	call LavaridgeTown_Gym_1F_EventScript_CheckHideTrainers
 	end
 
-LavaridgeTown_Gym_1F_EventScript_1FE705:: @ 81FE705
+@ Unclear where/if these temp vars are getting checked
+LavaridgeTown_Gym_1F_EventScript_SetTrainerTempVars:: @ 81FE705
 	setvar VAR_TEMP_B, 0
 	setvar VAR_TEMP_C, 0
 	setvar VAR_TEMP_D, 0
 	setvar VAR_TEMP_E, 0
 	setvar VAR_TEMP_F, 0
 	checktrainerflag TRAINER_COLE
-	goto_if_eq LavaridgeTown_Gym_1F_EventScript_1FE72C
+	goto_if_eq LavaridgeTown_Gym_1F_EventScript_SetGeraldTempVar
 	setvar VAR_TEMP_B, 1
 
```

Comment:
Let's remove these empty lines to better indicate the fall-through nature of these scripts.